src/Raven.Server/Documents/Handlers/Admin/AdminLogsHandler.cs

```diff
@@ -1,12 +1,54 @@
-﻿using System.Threading.Tasks;
+﻿using System;
+using System.Threading.Tasks;
+using Raven.Client.ServerWide.Operations.Logs;
+using Raven.Server.Json;
 using Raven.Server.Routing;
 using Raven.Server.Web;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
 using Sparrow.Logging;
 
 namespace Raven.Server.Documents.Handlers.Admin
 {
     public class AdminLogsHandler : RequestHandler
     {
+        [RavenAction("/admin/logs/configuration", "GET", AuthorizationStatus.ClusterAdmin)]
+        public Task GetConfiguration()
+        {
+            using (ServerStore.ContextPool.AllocateOperationContext(out JsonOperationContext context))
+            using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
+            {
+                var djv = new DynamicJsonValue
+                {
+                    [nameof(GetLogsConfigurationResult.CurrentMode)] = LoggingSource.Instance.LogMode,
+                    [nameof(GetLogsConfigurationResult.Mode)] = ServerStore.Configuration.Logs.Mode,
+                    [nameof(GetLogsConfigurationResult.Path)] = ServerStore.Configuration.Logs.Path.FullPath,
+                    [nameof(GetLogsConfigurationResult.UseUtcTime)] = ServerStore.Configuration.Logs.UseUtcTime
+                };
+
+                var json = context.ReadObject(djv, "logs/configuration");
+
+                writer.WriteObject(json);
+            }
+
+            return Task.CompletedTask;
+        }
+
+        [RavenAction("/admin/logs/configuration", "POST", AuthorizationStatus.ClusterAdmin)]
+        public async Task SetConfiguration()
+        {
+            using (ServerStore.ContextPool.AllocateOperationContext(out JsonOperationContext context))
+            {
+                var json = await context.ReadForMemoryAsync(RequestBodyStream(), "logs/configuration");
+
+                var configuration = JsonDeserializationServer.Parameters.SetLogsConfigurationParameters(json);
+
+                LoggingSource.Instance.SetupLogMode(configuration.Mode, Server.Configuration.Logs.Path.FullPath);
```

Comment:
Validate arguments here ?

Reply:
I'm not changing the path (it is the default one). Only the mode.